pkg/cmd/init.go

```diff
@@ -63,3 +67,26 @@ func ServerCommand(arg *args) *cobra.Command {
 		},
 	}
 }
+
+// ReadConfigCommand creates a new cobra.Command to load the calls config for Deriv API.
+// The config is loaded and then pushed to etcd for watching changes.
+// It can take cfgPath of type *string which is the path to the configuration file as an argument.
+// It also takes the etcd host URL and dial timeout in seconds as argument
+// It returns a pointer to a cobra.Command which can be executed to load the config.
+// It returns an error if the logger initialization fails, the configuration cannot be loaded, or there is error thrown by etcd.
+func ReadConfigCommand(arg *args) *cobra.Command {
+	return &cobra.Command{
+		Use:   "read-config",
+		Short: "Read config and push call config to etcd",
+		Long:  "Read config and push call config to etcd for hot reloads. Also sets up a watcher for the config",
```

Comment:
What do you think if we design commands like:
- `config upload`
- in future we can add `config backup` or `config download`